lib/tools/xctest-commands.js

```diff
@@ -0,0 +1,42 @@
+import { fixOutputToArray } from '../helpers';
+
+const xctestCommands = {};
+
+/**
+ * Runs an xcuitest on the device or simulator
+ * @param {string} testRunnerBundleId The bundle id of the test runner
+ * @param {string} appUnderTestBundleId The bundle id of the app under test
+ * @param {string} xctestBundleId The bundle id of the xctest package
+ * @returns {SubProcess}
+ */
+xctestCommands.runXCUITest = async function runXCUITest (testRunnerBundleId, appUnderTestBundleId, xctestBundleId) {
+  const uiTestProcess = this.createSubProcess([
+    'xctest', 'run',
+    'ui', xctestBundleId,
+    appUnderTestBundleId, testRunnerBundleId
+  ]);
+  await uiTestProcess.start(0);
+  return uiTestProcess;
+};
+
+/**
+ * Installs a xctest bundle on the device or simulator
+ * @param {string} xctestBundlePath the path of the xctest bundle
+ * @returns {string} the bundle id of xctest bundle that was installed
+ */
+xctestCommands.installXCTestBundle = async function installXCTestBundle (xctestBundlePath) {
+  const output = await this.exec(['xctest', 'install', xctestBundlePath, '--json']);
+  return JSON.parse(output).installedTestBundleId;
```

Comment:
I think it makes sense to cover `parse` call into try/catch, so we can print a meaningful error if the command output gets corrupted